src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-switcher.html

```diff
@@ -3,9 +3,9 @@
 <script>
 document.write(`
   <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="{{ _('light/dark') }}" aria-label="{{ _('light/dark') }}" data-bs-placement="bottom" data-bs-toggle="tooltip">
-    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
-    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
-    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
+    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light" title='Current theme: light'></i>
+    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark" title='Current theme: dark'></i>
+    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto" title='Current theme: follows OS Default'></i>
```

Comment:
Hmm... how about...

- button title "Appearance" or "Color scheme" or "Color mode" (I'm trying to avoid using the word "theme" since it is used in "PyData Theme")
  - title "Light"
  - title "Dark"
  - title "Auto (follow system settings)"

Reply:
+1 for "color mode" but I'll defer to Quansight's expertise here. In my head it's always called "light/dark mode".

Reply:
Hiya! I'm commenting since I was mentioned elsewhere on the PR.

I think it's worth noting that I think any of these options would work fine, including the ones @Carreau already has in the PR. The only one I'd be hesitant to recommend is "Appearance" because, while accurate, is more general than needed.

If I had to choose one to be a tie-breaker, I'd go with
- Button title: "Color mode"
- Current theme light: "Light"
- Current theme dark: "Dark"
- Current theme auto: "System Settings"

Hope that helps. @ me if you have any more questions about this!